rust/src/worker.rs

```diff
@@ -192,6 +192,8 @@ pub struct WorkerSettings {
     /// "WebRTC-Bwe-AlrLimitedBackoff/Enabled/".
     #[doc(hidden)]
     pub libwebrtc_field_trials: Option<String>,
+    /// Disable liburing (io_uring) despite it's supported in current host.
+    pub disable_liburing: Option<bool>,
```

Comment:
What about this instead?:
```suggestion
    pub disable_liburing: bool,
```

Or better yet (disable: false is much less natural):
```suggestion
    pub enable_liburing: bool,
```

Reply:
No please, here we want to keep current behavior and parity with Node layer. Default behavior must be that liburing is enabled if supported by current host.

And why should `disable_liburing` be mandatory? I want it to be optional.

Reply:
Because by making it optional:
* you have tri-state boolean: Some(true), Some(false), None
* you don't let user know what the default value is

By having it as `bool` API is much clearer and self-documenting. Setting option to `false` is just as easy as setting it to `None` in `impl Default for WorkerSettings`.

On worker level it will be the same, but on Rust API level I would prefer `enable_io_uring` defaulting to `true`, which user can set to `false`. This is a clearer API, doesn't surface what library is used for `io_uring` and it is more natural to read `enable: true|false`.

Reply:
> on Rust API level I would prefer enable_io_uring defaulting to true

It's not defaulting to anything, you are making it mandatory for the user to pass true or false.

Reply:
You may be misunderstanding Rust API. It is not making anything mandatory, the field always has to be specified anyway and you're setting it to `None` in `impl Default for WorkerSettings`. You can't not specify fields like in JavaScript. So you just change `Option<bool>` to `bool` and set default value to `false` instead of `None`.

The struct is intentionally marked as `#[non_exhaustive]`, so user can't construct it directly anyway, the only way is to go through APIs provided by the library like `impl Default for WorkerSettings`. This is also the reason why this is actually **not** a breaking change for library API!

Reply:
ok done: https://github.com/versatica/mediasoup/pull/1442/commits/0529246e7668f97542bf850e386434cd5ca6012b